types/finding.ts

```diff
@@ -83,6 +83,20 @@ export interface FindingDeleteRequest {
   emailAddresses: string[];
 }
 
+export interface AwardFinding {
+  contest: number;
+  handle: string;
+  finding: string;
+  risk: string;
+  score: number;
```

Comment:
To support the historical data set this will need to be:

```suggestion
  score: number | null;
```